scalding-core/src/main/scala/com/twitter/scalding/Operations.scala

```diff
@@ -467,10 +467,10 @@ package com.twitter.scalding {
     }
 
     def complete(flowProcess: FlowProcess[_], call: AggregatorCall[X]): Unit = {
-      emit(flowProcess, call)
+      emit(call)
     }
 
-    def emit(flowProcess: FlowProcess[_], call: AggregatorCall[X]): Unit = {
+    def emit(call: AggregatorCall[X]): Unit = {
       call.getOutputCollector.add(set(call.getContext))
```

Comment:
I hesitated to inline the method because I am not sure if the code is used by any client code, as `com.twitter.scalding.FoldAggregator.emit` seems to be a public method.

Moreover, the original emit method could have been inlined as well, but hadn't. So I guess there's a good reason for it.